war/pom.xml

```diff
@@ -587,17 +599,42 @@ THE SOFTWARE.
     <profile>
         <id>node-classifier-linux</id>
         <activation><os><family>Linux</family><arch>amd64</arch></os></activation>
-        <properties><node.download.file>node-v${node.version}-linux-x64.tar.gz</node.download.file></properties>
+		<properties>
+			<node.download.file>node-v${node.version}-linux-x64.tar.gz</node.download.file>
+			<node.download.classifier></node.download.classifier>
+		</properties>
```

Comment:
Did something change on these or was it just a formatting change? Makes it hard to read the diff.